packages/framework/src/Model/Advert/Exception/AdvertPositionNotKnownException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Model\Advert\Exception;
+
+use Exception;
+
+class AdvertPositionNotKnownException extends Exception implements AdvertException
+{
+    public function __construct(string $positionName, array $knownPositionsNames)
+    {
+        $message = sprintf(
+            'Unknown advert position name: "%s". These are known names: %s.',
```

Comment:
Quotes are missing around the second placeholder.

Also, the formulation is a bit dry. How about `Unknown advert position name "%s". Known names are "%s".`?

Reply:
quotes are missing because I want to quote each name separately :slightly_smiling_face: I will change the formulation though :+1: